### PR TITLE
Build promise chain, themes/custom, and var declarations.

### PR DESCRIPTION
**Pull request by chasingmaxwell**
**Changes:**
- Correctly chain everything together in the build process.
- Delete the current build before creating a new one.
- Create all the appropriate symlinks.
- Convert the entire codebase to a single var declaration with aligned operators style.

**To Review:**
1. Create a new project.
2. Build that project.
3. Build it again.

### CODE REVIEW

File: lib/build.api.js [hunk 1]
```diff
@@ -38,18 +39,52 @@ module.exports = function (Aquifer) {
...
-          //fs.symlinkSync(path.join(self.destination, 'sites/all/modules/contrib'), self.project.absolutePaths.modules.contrib, 'directory');
-          //fs.symlinkSync(path.join(self.destination, 'sites/all/modules/custom'), self.project.absolutePaths.modules.contrib, 'directory');
-          //fs.symlinkSync(path.join(self.destination, 'sites/all/modules/features'), self.project.absolutePaths.modules.features, 'directory');
+      // Initialize drush and the build promise chain.
+      drush.init()
+        // Delete current build.
+        .then(function () {
+          var def = new Deferred();
+
+          fs.remove(self.destination, function (err) {
```
Comment (patrickocoffeyo) [inline lib/build.api.js]: Can we just use fs.removeSync? We're deferring this but we don't need to do that if we just use the sync version of this function.

**Comment by patrickocoffeyo** [comment]
@chasingmaxwell this is super awesome. Could you possibly create an editorconfig file with the right colorcolumn config to help us write consistent assignments?

**Comment by chasingmaxwell** [comment]
@patrickocoffeyo I was looking for a way to enforce the indentation and alignment of operators, but colorcolumn doesn't appear to be it since you can only set it to a single value. The alignment of operators will be variable depending on the length of the variables in the var declaration.

**Comment by patrickocoffeyo** [comment]
This looks great! :shipit:

**Pull request merged**